administration/src/bp-modules/applications/JsonFieldElemental.tsx

```diff
@@ -96,7 +97,7 @@ const JsonFieldElemental = ({
     case 'String':
       return (
         <p>
-          {t(getTranslationKey())}: {jsonField.value}
+          {t(getTranslationKey())}:{EmailLink(jsonField.value)}
```

Comment:
:question: it seems a bit confusing to me that we return an “EmailLink” component for each field with type String.
But I don't have a better suggestion though. Maybe someone else could offer some advice here.

Reply:
Maybe introduce new type of the field - email

Reply:
i think a new field type is not that easy to implement since this type is stored in the database and has to backward compatible.

According the naming i think i agree with tory.
I think its cleaner to create a "MailLink" component that is a clickable anchor and don't put that `isEMailValid` logic inside of the component but keep it dump.
So this component in any case delivers what it says
You can keep that logic in the `JsonFieldElemental` component